# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../tdb/hotfix.patch
# Copyright (C) 2014 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff --git a/include/tdb.h b/include/tdb.h
index 5ea5e60..03e429c 100644
--- a/include/tdb.h
+++ b/include/tdb.h
@@ -31,6 +31,7 @@ extern "C" {
 #endif

 #include <signal.h>
+#include <stdbool.h>

 /**
  *  <at> defgroup tdb The tdb API
